examples/aws/terraform/starter-cluster/ssm.tf

```
@@ -5,4 +5,5 @@ resource "aws_ssm_parameter" "license" {
   type      = "SecureString"
   value     = file(var.license_path)
   overwrite = true
+  tier      = "Advanced"
```

Ditto previous comment

```suggestion
  tier      = "Intelligent-Tiering"
```